examples/sentiment/train_sentiment.py

```diff
@@ -118,10 +118,18 @@ def evaluate(model, test_trees):
         acc_root, result['correct_root'], result['total_root']))
 
 
+@chainer.dataset.converter()
+def convert(batch, _):
+    return batch
```

Comment:
It's not related to the PR but I just wonder why `device` can be omitted. Do you know anything?

Reply:
The batch contains `str`s actually, not ndarrays.

Reply:
Hmm, I see. thanks.